### PR TITLE
[Improvement] add pulsar metrics

### PR DESCRIPTION
**Pull request by chyezh**
#26906

### CODE REVIEW

**Comment by chyezh** [comment]
After this PR, metrics of pulsar client side will be registered into milvus if `pulsar.enableClientMetrics=true`.
image

File: pkg/util/paramtable/service_param.go [hunk 1]
```diff
@@ -615,6 +618,14 @@ func (p *PulsarConfig) Init(base *BaseTable) {
 		Export:       true,
 	}
 	p.RequestTimeout.Init(base.mgr)
+
+	p.EnableClientMetrics = ParamItem{
+		Key:          "pulsar.enableClientMetrics",
+		Version:      "2.3.0",
```
Comment (czs007) [inline pkg/util/paramtable/service_param.go:624]: better to be 2.3.2

**Pull request merged**